### DISCUSSION TITLE
Paulmann is incorrectly detected

### DISCUSSION THREAD

**Discussion by Fasj2001** [Koenkk/zigbee2mqtt#18868] [Devices]
Hello,
I have integrated a Paulmann 93999 (https://www.zigbee2mqtt.io/devices/93999.html).
but it is incorrectly recognized as a lamp (https://www.zigbee2mqtt.io/devices/500.45.html).
Works so far, except that the lamps that the 93999 controls from time to time flash briefly.
Several times tried to re-integrate, did not work.
What can I do?
Thanks!

**Comment by Koenkk** [comment]
Can you provide the data/database.db entry of this device?

**Reply by Fasj2001** [reply]
Not sure :-(
Hope you mean this:
{"id":2,"type":"Router","ieeeAddr":"0x00158d0008431301","nwkAddr":52648,"manufId":4151,"manufName":"Paulmann Licht","pow                                                                                    erSource":"Mains (single phase)","modelId":"500.45","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":257                                                                                    ,"inClusterList":[0,4,3,6,8,5,4096],"outClusterList":[25],"clusters":{"genOnOff":{"attributes":{"onOff":0}},"genBasic":{                                                                                    "attributes":{"zclVersion":3}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId"                                                                                    :97,"inClusterList":[33],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersio                                                                                    n":16,"stackVersion":1,"hwVersion":1,"dateCode":"20190515","swBuildId":"1400-0001","zclVersion":3,"interviewCompleted":t                                                                                    rue,"meta":{"configured":88764544},"lastSeen":1693935763231,"defaultSendRequestWhen":"immediate"}

If not, please give me a hint, how i get the Info.

**Reply by Koenkk** [reply]
It seems your device came with the wrong modelId: `"modelId":"500.45"`, this is an issue on the device side. The only thing you can do is update it manually:
1. stop z2m
2. Change `"modelId":"500.45"` to `"modelId":"H036-0001"` in the data/database.db
3. Start z2m

**Reply by Fasj2001** [reply]
Thank´s !
It works.

**Reply by Biker1989** [reply]
Kann mir einer erklären wie ich das Manuell umschreibe?
